github/data_source_github_release.go

```diff
@@ -108,8 +109,38 @@ func dataSourceGithubReleaseRead(d *schema.ResourceData, meta interface{}) error
 
 	var err error
 	var release *github.RepositoryRelease
-
 	switch retrieveBy := strings.ToLower(d.Get("retrieve_by").(string)); retrieveBy {
+	case "latest_prerelease":
+		// The GitHub API doesn't specify a way to just load the most recent prerelease,
+		// so we'll load all releases and determine which one is the most recent
+		// prerelease after.
+		log.Printf("[INFO] Refreshing GitHub latest prerelease from repository %s", repository)
+		var releases []*github.RepositoryRelease
+		nextPage := 1
+		// TODO: 10 is sort of arbitrary here -- what's the best way to allow configurability
+		// for this to prevent the provider from becoming glacially slow on large repos?
```

Comment:
This is an excellent question. Do you know what the max number of releases we can retrieve per page with the API?

Reply:
100 is the limit.